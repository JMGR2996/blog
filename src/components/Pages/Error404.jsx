import React from 'react'

const Error404 = () => (
  <div className='l-container container'>
    Error 404
  </div>
)

export default Error404
