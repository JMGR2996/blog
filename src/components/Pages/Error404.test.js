import React from 'react'
import { mount, configure } from 'enzyme'
import Error404 from './Error404'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })

describe('Render Eror 404', () => {
  it('render correctly Error 404 page', () => {
    const Error404Component = mount(<Error404 />)
    expect(Error404Component).toMatchSnapshot()
  })
})
