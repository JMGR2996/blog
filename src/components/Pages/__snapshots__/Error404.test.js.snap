// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Eror 404 render correctly Error 404 page 1`] = `
<Error404>
  <div
    className="l-container container"
  >
    Error 404
  </div>
</Error404>
`;
